addons/web/controllers/main.py

```diff
@@ -1057,6 +1057,23 @@ def image(self, model, id, field, **kw):
             image_data = self.placeholder()
         headers.append(('ETag', retag))
         headers.append(('Content-Length', len(image_data)))
+
+        # Guess mime type
+        signatures = {
+            "image/jpeg": ['\xFF\xD8\xFF\xE0', '\xFF\xD8\xFF\xE2',
+                           '\xFF\xD8\xFF\xE3', '\xFF\xD8\xFF\xE1'],
+            "image/gif": ['GIF87a', 'GIF89a'],
```

Comment:
That would make sense https://github.com/odoo/odoo/issues/12627 gets fixed, and that fix won't come in v8 (if it ever comes), but until then, if you cannot upload SVG, there is little chance that you are downloading it, don't you think? :wink: